### ISSUE TITLE
[Edit] - "Functions"-Chapter

### ISSUE THREAD

**Issue by henningsway** [Tazinho/Advanced-R-Solutions#101]
## Notes and Questions (from the editing process)
- lets try to keep track of the "open ends" here

**Comment by henningsway** [comment]
phew, this is a long (and important) chapter. Had an initial look. Let me know, when you had your initial look. :)

(no hurry!)

**Comment by Tazinho** [comment]
Yep, I suggest we edit the first (or first 3) subsections and then do the rest. Was a bit busy this week, but will try to make some progress ASAP.

**Comment by Tazinho** [comment]
@henningsway you could already start with the first two subsections (except for the very first exercise). As it turns out almost every exercise of this chapter needs some serious editing and therefore I might take quite some time before I can hand over the other subsections to you.

**Comment by Tazinho** [comment]
@henningsway: I am mostly done here. I think it's time for you now. Afterwards let's try to resolve as much of possible of the remaining.

**Comment by henningsway** [comment]
Alright, I've gone through the entire chapter and improved and clarified the answers as much as possible. The three orange exercises are still orange, other than that it's pretty good - I'd think.

I propose closing this issue for now and potentially come back to these answers once we are done with everything else. :balloon: :100: :)
